From 64e63ad756bf0fe539a476302b994f5a2b7afc87 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Wed, 21 Jun 2023 14:38:38 +0200
Subject: [PATCH 0412/1141] ARM: configs: disable MTD_SPI_NOR_USE_4K_SECTORS on
 fragment-01-defconfig_cleanup

This allows to avoid to limit the erase block size to 4K.
Signed-off-by: Patrice Chotard <patrice.chotard@st.com>
Change-Id: Iaf215e52c524574efed29ed9be18b6e16ea4473f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/319076
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/configs/fragment-01-defconfig-cleanup.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-01-defconfig-cleanup.config b/arch/arm64/configs/fragment-01-defconfig-cleanup.config
index bdaf6e42e1f5..9e39d7422c60 100644
--- a/arch/arm64/configs/fragment-01-defconfig-cleanup.config
+++ b/arch/arm64/configs/fragment-01-defconfig-cleanup.config
@@ -45,3 +45,4 @@
 # CONFIG_MEDIA_SDR_SUPPORT is not set
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 # CONFIG_MTD_PHYSMAP is not set
+# CONFIG_MTD_SPI_NOR_USE_4K_SECTORS is not set
-- 
2.39.2

